index.bs

```diff
@@ -345,8 +345,7 @@ await cookieStore.set({
   path: '/',
 
   // Creates secure cookies by default on secure origins.
-  secure: true,
-  httpOnly: false,
+  secure: true
```

Comment:
nit: Would you be OK with retaining the trailing comma, to minimize future diffs?

(I can imagine that new devs would be a bit confused by this, so it's a trade-off)

Reply:
Yeah, can do.